scripts/test-harness-contracts.ps1

```
@@ -1486,6 +1486,23 @@ try {
 }
 
+# ---------------------------------------------------------------------------
+# Issue #76 — mutation is a self-consistency check, structurally blind to "this code solves the
+# wrong problem": the real boot defect survived 4 adversarial rounds and every mutation and was
+# caught by reading the sibling predicate. Three facets, three homes.
+# ---------------------------------------------------------------------------
+try {
+    Check-Match '.claude/skills/pelizzai-review/references/code-reviewer.md' 'The sibling check \(a distinct step' 'code-reviewer: the sibling check is a named step'
+    Check-Match '.claude/skills/pelizzai-review/references/code-reviewer.md' 'LOCATE the other place\(s\) in the\s+repository that solve the same problem' 'code-reviewer: guards/predicates are compared with their siblings'
+    Check-Match '.claude/skills/pelizzai-review/references/code-reviewer.md' 'structurally blind\s+to .this code solves the wrong problem.' 'code-reviewer: names the blindness of checking the code against itself'
+    Check-Match '.claude/skills/pelizzai-tdd/SKILL.md' 'A predicate with N conditions demands N scenarios, one per condition' 'tdd: N conditions demand N scenarios'
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Check-Match' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)